packages/teams-js/src/internal/uuidObject.ts

```diff
@@ -0,0 +1,17 @@
+import { generateGUID, validateUuid } from './utils';
+
+/**
+ * @internal
+ * Limited to Microsoft-internal use
+ *
+ * UUID object
+ */
+export class UUID {
+  public constructor(public readonly uuid: string = generateGUID()) {
```

Comment:
>public [](http://example.com/codeflow?start=21&length=6)

Can you say more about why this needs to be `public` now? Enabling direct access to the underlying data defeats the abstraction provided by the class.

Reply:
Sorry, this shouldn't be public, That was just used for testing purposes. Good catch! Changing it back to private